src/Core/Types.Filters/Expressions/OperationHandlers/ExpressionOperationHandlers.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace HotChocolate.Types.Filters.Expressions
 {
-    internal static class ExpressionOperationHandlers
+    public static class ExpressionOperationHandlers
```

Comment:
@michaelstaib  I opened this API. This way it's easier to reference all operations handlers in the comming conventions